src/ir/effects.h

```diff
@@ -472,6 +480,24 @@ class EffectAnalyzer {
       self->parent.catchDepth--;
     }
 
+    static void doStartTryTable(InternalAnalyzer* self, Expression** currp) {
+      TryTable* curr = (*currp)->cast<TryTable>();
```

Comment:
```suggestion
      auto* curr = (*currp)->cast<TryTable>();
```

Reply:
(Looks like the code above that this is based on has the same issue, but let's add the new code in the proper form. We can separately improve the other code - if you feel like it, that could be done in this PR too.)